{{cookiecutter.repo_name}}/README.rst

```diff
@@ -1,23 +1,150 @@
 README for {{cookiecutter.project_desc}}
 
+
 Testing with Docker
 -------------------
 
 This XBlock comes with a Docker test environment ready to build, based on the xblock-sdk workbench. To build and run it::
 
-        $ make dev.run
+    $ make dev.run
 
 The XBlock SDK Workbench, including this XBlock, will be available on the list of XBlocks at http://localhost:8000
 
 Translating
 -----------
 
+Internationalization (i18n) is when a program is made aware of multiple languages.
+Localization (l10n) is adapting a program to local language and cultural habits.
+
 Use the locale directory to provide internationalized strings for your XBlock project.
+For more information on how to enable translations, visit the
+Open edX XBlock tutorial on Internationalization <http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html>.
```

Comment:
It seems the hyperlinks aren't rendering correctly, [check it out](https://github.com/edx/cookiecutter-xblock/blob/a454746b763b72df57ad891f3f80d4415048c3bd/%7B%7Bcookiecutter.repo_name%7D%7D/README.rst)

Found some docs on the syntax [here](http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets).

Reply:
@viadanna Ah, good catch.
Fixed it. Thanks :)